src/libstd/path/windows.rs

```diff
@@ -87,7 +87,7 @@ pub struct Path {
 
 impl fmt::Show for Path {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "Path {{ {} }}", self.display())
```

Comment:
I think we'll want to tweak the exact `Show` implementations over time, this may not even be what we want to end up at. We may want, for example, to render non-utf8 characters with some form of escape or something. For now though I just wanted to keep this consistent with `display()` to smooth migrations.